### DISCUSSION TITLE
Help: Github Copilot doesn't work on my VSCode

### DISCUSSION THREAD

**Discussion by xareelee** [community/community#52117] [Copilot Conversations]
### Select Topic Area

Bug

## What did I do

I just joined and tried GitHub Copilot. I followed the [quickstart for GitHub Copilot](https://docs.github.com/en/copilot/quickstart). I tried to see my first suggestion by typing JS code (following the tutorial):

```js
function calculateDaysBetweenDates(begin, end) {
```

Issues I encountered:

- By typing every char on my own (not copy-and-paste), VSCode autocompleted my code with close curly bracket at the end of line:  `function calculateDaysBetweenDates(begin, end) {}`, and it did not show any suggestion when typing code.
- When I delete the close curly bracket, it did not show any suggestion. I checked the GitHub Copilot Output, but it showed nothing.
- When I tried to troubleshoot this issue and file this issue here (this takes 30 minutes), it now finally shows the suggestion (without changing anything).
- Now I re-type the code again, and it shows suggestions. However, when I pressed `option + ] (macOS)` to see alternative suggestions, VSCode print `‘` symbol on my cursor (which is a macOS keyboard input).
- After a couple of minutes, I tried to type the same function and pressed `option + ] (macOS)` again. Now it works (without changing anything).

My first experience of using an GitHub Copilot was very bad. I feel frustrated.

My questions:

- Why this happened?
- Is there any way to avoid this happen again?

**Comment by zhou-ziyan** [comment]
I encountered the exactly same error minutes ago.
I tried using shortcut `shift + cmd + P`, found the 'Github Copilot: Open Completions Panel' option, and selected it by pressing Enter.
This made the suggestions start appearing, so it seems to have worked! I can't guarantee it will work in your environment, but you could give it a try.

**Reply by xareelee** [reply]
I think there should be a shortcut in VSCode to make the GitHub Copilot extension to show a popup which shows the best in-line suggestion and the working status (error messages or hints). Otherwise, I won't know what happen to GitHub Copilot.
